### PR TITLE
test: add mgr section to the host inventory file

### PR DESCRIPTION
**Pull request by leseb**
Without this, we don't test the mgr role so we need to add it.

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
jenkins test kraken-ansible2.2-dmcrypt_journal

**Comment by guits** [comment]
jenkins test kraken-ansible2.2-dmcrypt_journal

**Comment by guits** [comment]
jenkins test jewel-ansible2.2-update_cluster

**Comment by guits** [comment]
jenkins test jewel-ansible2.2-docker_dedicated_journal

**Comment by guits** [comment]
jenkins test kraken-ansible2.2-purge_cluster

**Comment by guits** [comment]
jenkins test jewel-ansible2.2-docker_dedicated_journal

**Comment by guits** [comment]
jenkins test jewel-ansible2.2-docker_dedicated_journal

**Comment by guits** [comment]
@leseb I fixed this PR.
When you have a moment to take a look :)

File: infrastructure-playbooks/rolling_update.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@
     - "{{ osd_group_name|default('osds') }}"
     - "{{ mds_group_name|default('mdss') }}"
     - "{{ rgw_group_name|default('rgws') }}"
+    - "{{ mon_group_name|default('mgrs') }}"
```
Comment (leseb) [inline infrastructure-playbooks/rolling_update.yml]: `mon_group_name`?

**Comment by guits** [comment]
jenkins test jewel-ansible2.2-centos7_cluster

**Comment by guits** [comment]
jenkins test jewel-ansible2.2-purge_cluster

**Comment by guits** [comment]
jenkins test Docs

**Comment by guits** [comment]
jenkins test kraken-ansible2.2-centos7_cluster

**Comment by leseb** [comment]
@guits thanks!

**Pull request merged**